### DISCUSSION TITLE
paperless pre_consume script

### DISCUSSION THREAD

**Discussion by ciB89** [alexbelgium/hassio-addons#816] [Q&A] [answered]
I tried adding a pre_consume script, but whatever I try logs keep telling me, that the files could not be found.  I double checked, the files are available to/in the container addon. Reading the documentation though, it seems that I specfically need to mount the folder to the webserver container, which might be the problem why it is not working. 

Any help on this?

**Comment by ciB89** [comment]
Okay, this is really weird. I tried a different location (in /share) and it seemed to work because now the logs at least showed "permission denied" for the file. I realized I hadn't added +x to the files, so I did that and restarted the addon. But now I get the same old error again:

`[2023-04-29 19:41:32,332] [ERROR] [celery.app.trace] Task documents.tasks.consume_file[d976a4ba-88c6-4fab-b16b-d8cd4ab4b820] raised unexpected: ConsumerError("test.pdf: Error while executing pre-consume script: [Errno 2] No such file or directory: '/share/paperless/scripts/pre_consume.sh'")`

**Comment by ciB89** [comment, marked as answer]
Man, I feel stupid, I unfortunately had not changed the editor to LF (unix) when edititng the files on my windows machine.. it works now.

**Reply by alexbelgium** [reply]
Thanks for the feedback! Glad it works
